components/picker/metadata/picker.yml

```diff
@@ -103,59 +106,76 @@ examples:
 
         <div class="dummy-spacing" style="block-size: 15rem;"></div>
 
+        <!-- Side Label -->
+        <div class="spectrum-Examples-item">
+          <h4>Side Label</h4>
+          <div class="spectrum-FieldLabel spectrum-FieldLabel--sizeM spectrum-Picker--sideLabel spectrum-FieldLabel--left">Country</div>
+          <div style="display: inline-block">
```

Comment:
You mentioned above that the Picker will need to be wrapped in a `div` that has `inline-flex` applied to it, but in these examples you're using `inline-block`. It looks like it works...but I was wondering if you had a typo?

Reply:
Yep I did! It should be `inline-block` in the description. SWC uses `inline-flex` which is where I probably got confused, but they're adding some additional styling for the popover positioning. For it to look ok out of the box, it should be `inline-block`

Reply:
"need" here feels like we missed something. If we intend to require the `spectrum-Picker--sideLabel` modifier, why can't this also add `display: inline-flex` as well?

Reply:
"need" might be too strong of a word but `inline-flex` throws off the transform that is happening to get the Popover to display below the Picker since it no longer starts on a "new line" with flex. There's a way to do this without that div and only use the `spectrum-Picker--sideLabel` class but it would not be a pure CSS solution, I would have to throw in some javascript in order to get that menu to align properly with the Picker. Unless you all have an idea I didn't think of!

The `spectrum-Picker--sideLabel` class was added in order to have a way to identify when the side label layout was being used from a component level, specifically for the Quiet variant, since styling for the entire component needed to change based on it. 

<img width="348" alt="Screenshot 2023-07-11 at 9 16 02 AM" src="https://github.com/adobe/spectrum-css/assets/99203545/b044abe4-99fa-46a2-931f-58c87cb6df99">